templates/privatebin/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Output,
+  Services,
+  randomPassword,
+  randomString,
+} from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const databasePassword = randomPassword();
+  const randomKey = randomString();
+
+  services.push({
+    type: "app",
+    data: {
+      serviceName: input.appServiceName,
+      env: [].join("\n"),
+      source: {
+        type: "image",
+        image: input.appServiceImage,
+      },
+      domains: [
+        {
+          host: "$(EASYPANEL_DOMAIN)",
+          path: "/",
+          port: 80,
+        },
+      ],
+      mounts: [
+        {
+          type: "volume",
+          name: "data",
+          mountPath: "/srv/data",
+        },
+        {
+          type: "file",
+          content:
```

Comment:
move this to a multi line string in a const

Reply:
@deiucanta I have fixed that and tested.